app/js/controllers/foyer/individuForm.js

```diff
@@ -37,38 +36,35 @@ angular.module('ddsApp').controller('FoyerIndividuFormCtrl', function($scope, in
         }
     ];
 
-    $scope.individu = {
-        nationalite: 'fr',
+    var defaultIndividu = {
+        nationalite: 'ue',
         assPreconditionRemplie: false,
         scolarite: 'college',
         tauxInvalidite: 'moins50',
         boursier: false,
         aCharge: true,
         place: false,
-        role: individuRole
+        titreSejour10ans: true,
+        titreSejour5ans: true,
+        role: individuRole,
+        situationsPro: []
     };
 
-    if (true === ($scope.captureRelationConjoint = !!options.captureRelationConjoint)) {
-        $scope.individu.statutMarital = 'mariage';
-    }
-
+    var individu = $scope.individu = _.find($scope.situation.individus, { role: individuRole }) || defaultIndividu;
     var isIndividuParent = IndividuService.isRoleParent(individuRole);
 
-    if (isIndividuParent) {
-        var individu = _.find($scope.situation.individus, { role: individuRole });
-        if (individu) {
-            $scope.individu = _.merge($scope.individu, individu);
-            $scope.individu.situationsPro.forEach(function(situationPro) {
-                $scope.selectedStatuts[situationPro.situation] = true;
-            });
-        }
+    individu.situationsPro.forEach(function(situationPro) {
+        $scope.selectedStatuts[situationPro.situation] = true;
+    });
+
+    if (options.captureRelationConjoint) {
+        $scope.individu.statutMarital = 'mariage';
```

Comment:
Be consistent with usage of `$scope.individu` vs local `individu` being exported to `$scope.individu` at some point.